hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/UpgradeDowngrade.java

```diff
@@ -144,16 +154,12 @@ public void run(HoodieTableVersion toVersion, String instantTime) {
               .run(toVersion, instantTime);
         }
       } catch (Exception e) {
-        LOG.warn("Unable to upgrade or downgrade the metadata table to version " + toVersion
-            + ", ignoring the error and continue.", e);
+        throw new HoodieUpgradeDowngradeException("Upgrade/downgrade for the Hudi metadata table failed. "
```

Comment:
can we write a test for this case.

Reply:
Sure will look into adding this

Reply:
ok. did you get past the issue?

Reply:
This will be addressed eventually in the test fixtures PR https://github.com/apache/hudi/pull/13669. There is limitations of doing this validation in the current form of this test class.

Reply:
Fixtures PR has landed, have added the test here https://github.com/apache/hudi/pull/13642/files#r2253397677